pyproject.toml

```diff
@@ -111,6 +111,7 @@ dependencies = [
     "jinja2==3.1.4",
     "scikit-learn==1.5.1",
     "tqdm==4.66.5",
+    "starlett==0.37.2",
```

Comment:
was it added w/o updating lockfile?

Reply:
btw where it package came from?

Reply:
We somehow ended up with 
```
starlette==0.38.6 \
    --hash=sha256:4517a1409e2e73ee4951214ba012052b9e16f60e90d73cfb06192c19203bbb05 \
    --hash=sha256:863a1588f5574e70a821dadefb41e4881ea451a47a3cd1b4df359d4ffefe5ead
```
in requirements.txt, which caused
```
    |   File "/app-root/ols/app/main.py", line 127, in log_requests_responses
    |     response_body = response.body
    |                     ^^^^^^^^^^^^^
    | AttributeError: '_StreamingResponse' object has no attribute 'body'
```
at run time.

Reply:
>was it added w/o updating lockfile?

Yes, I missed that part. I suggest going ahead with this PR and fixing the lock file later, as pdm is not used in the build.